ca_description/urdf/create_base.xacro

```diff
@@ -54,6 +55,16 @@
     <xacro:create_wheel prefix="right" y_offset="${wheel_separation / -2}" wheel_radius="${wheel_radius}" wheel_width="${wheel_width}"/>
     <xacro:caster_wheel/>
 
+    <xacro:light_sensor prefix="left">
+      <origin xyz="0.095 0.14 0.065" rpy="0 ${pi/4} 0"/>
+      <optical_origin xyz="0.042 0 0" rpy="${-pi/2} 0 ${-pi/2}"/>
```

Comment:
Check the `optical_origin` block in both cases, does something change?

Reply:
Think about it in this way:

If you move your light sensor around, does the position move w.r.t. the sensor itself?

Reply:
I thought about this. But I thought also its maybe better to let the user of this model change the optical frame as he wishes without necessity of changing the model xacro file and (possibly) causing some undesired behaviour. Isn't this considered good practice ?

Reply:
Since you're modeling one specific model, the user shouldn't be able to modify that parameter in particular. I mean, you're modeling the LEGO one, not another.

Reply:
Ok. Makes sense. I'll change it